plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/server/LocalAppEngineServerBehaviour.java

```diff
@@ -139,8 +139,6 @@ void startDebugDevServer(List<File> runnables, MessageConsoleStream stream, int
     // Create run configuration
     DefaultRunConfiguration devServerRunConfiguration = new DefaultRunConfiguration();
     devServerRunConfiguration.setAppYamls(runnables);
-    // we can only connect with a single JVM
-    devServerRunConfiguration.setMaxModuleInstances(1);
```

Comment:
@patflynn's question during today's meeting makes me think that this should be kept until we have some UI to allow it to be set by the user in the launch config or the server config.